qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/DesiredNodesUpgradeIT.java

```diff
@@ -38,10 +38,7 @@ private enum ProcessorsPrecision {
     }
 
     public void testUpgradeDesiredNodes() throws Exception {
-        // Desired nodes was introduced in 8.1
-        if (UPGRADE_FROM_VERSION.before(Version.V_8_1_0)) {
-            return;
-        }
+        assumeFalse("Desired nodes was introduced in 8.1", UPGRADE_FROM_VERSION.before(Version.V_8_1_0));
```

Comment:
Maybe it's just me but I struggle to parse this, could we say this instead?

```suggestion
        assumeTrue("Desired nodes was introduced in 8.1", UPGRADE_FROM_VERSION.onOrAfter(Version.V_8_1_0));
```